### PR TITLE
	Control Bar Unit Tests

### PR DESCRIPTION
**Pull request by hghazzi**
### This PR will...

Add unit tests to handle updating buttons in the control bar

### Why is this Pull Request needed?

To improve code coverage of new code

### CODE REVIEW

**Comment by johnBartos** [comment]
@hghazzi There is no more right group. See my PR for what I've done with the dock buttons (they're inserted after the spacer).

File: test/unit/controlbar-test.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
+import ControlBar from 'view/controls/controlbar';
+import SimpleModel from 'model/simplemodel';
+import _ from 'test/underscore';
+import sinon from 'sinon';
+
+const model = _.extend({}, SimpleModel);
+// model.change = sinon.mock(_.extend({}, SimpleModel));
```
Comment (egreaves) [inline test/unit/controlbar-test.js]: Is this line of commented out code needed?

**Comment by hghazzi** [comment]
test this please

**Comment by egreaves** [comment]
@hghazzi do these tests need to be updated now that https://github.com/jwplayer/jwplayer/pull/2173 was merged?

**Comment by hghazzi** [comment]
test this please

**Comment by hghazzi** [comment]
Rebased and updated tests

**Pull request merged**